### PR TITLE
Compilation Parameters Frontend

### LINKED ISSUE #6: Compilation parameters (frontend)
- [x] Finalize design

### Design (work in progress)
Idea 1 
- Menu with Selects
- https://material-ui-next.com/demos/menus/#selected-menus

Idea 2
- Dialog
- https://material-ui-next.com/demos/dialogs/

Idea 3
- Settings drawer for compilation parameters
- Settings button left of Header
  - On click: open drawer
- Drawer contains multiple dropdowns for parameters

### Reference
- Compilation parameters: http://coconut.readthedocs.io/en/master/DOCS.html#setup

### PR DESCRIPTION
**Pull request by mjenrungrot**
To resolve #6 
- [x] Run button
- [x] Right-side's header
- [x] Prep for PR #11 (Issue #7)

image

image

image

### CODE REVIEW

**Comment by mjenrungrot** [comment]
Still have to fix tests. Because I removed Header component.

File: .eslintrc [hunk 1]
```diff
@@ -16,7 +16,13 @@
     "jest": true
   },
   "rules": {
+    "flowtype/no-types-missing-file-annotation": 0,
```
Comment (jcruz) [inline .eslintrc]: Remove

File: public/index.html [hunk 2]
```diff
@@ -8,6 +8,8 @@
       manifest.json provides metadata used when your web app is added to the
       homescreen on Android. See https://developers.google.com/web/fundamentals/engage-and-retain/web-app-manifest/
     -->
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
```
Comment (jcruz) [inline public/index.html]: Move to `src/index.js`

**Review by jcruz — APPROVED** [review]
LGTM

**Pull request merged**